### DISCUSSION TITLE
Can you price sqllite database and store the pictures attached in the code?

### DISCUSSION THREAD

**Discussion by HeliusHao** [massCodeIO/massCode#122] [General]
Can you price sqllite database and store the pictures attached in the code?

![image](https://user-images.githubusercontent.com/33107305/178201474-90a3c7e9-8db9-4f54-bc5c-1b21ac1b73b4.png)

**Comment by antonreshetov** [comment]
This is different from the ideas of the application - a simple snippet manager

**Reply by HeliusHao** [reply]
Sometimes it is necessary to keep the results of the feedback on the screen in the form of pictures, which will be more detailed, so this is more necessary

**Reply by antonreshetov** [reply]
Nevertheless, I do not plan to implement the attachments
